src/app/pages/home/containers/home-page.component.ts

```diff
@@ -22,6 +23,7 @@ import { HomePageAnimations } from './home-page.animation'
   animations: HomePageAnimations
 })
 export class HomePageComponent implements OnDestroy {
+  PROJECT_NAME = DefaultProjectName
```

Comment:
Project name can be confusing with the MP projects.
Maybe we can call it RADAR_BASE_INSTANCE or if you can think of something better?

Reply:
Yes, that would be better.